networking/ovn_kubernetes_network_provider/rollback-to-openshift-sdn.adoc

```diff
@@ -0,0 +1,10 @@
+[id="rollback-to-openshift-sdn"]
+= Rollback to the OpenShift SDN network provider
+include::modules/common-attributes.adoc[]
+:context: rollback-to-openshift-sdn
+
+toc::[]
+
+As a cluster administrator, you can rollback to the OpenShift SDN network provider from the OVN-Kubernetes network provider.
```

Comment:
We shall tell users that the rollback operation shall only be taken if the migration to OVN doesn't go well.

Reply:
@pliurh Shall we need note that it is better to do the rollback within some time ( maybe a couple of hours) if the migration to OVN doesn't go well?